src/promptflow-devkit/promptflow/_sdk/_utils.py

```diff
@@ -1000,13 +1010,20 @@ def create_temp_flex_flow_yaml(entry: Union[str, PathLike, Callable], code: Path
     flow_yaml_path = code / FLOW_FLEX_YAML
     existing_content = None
 
+    if flow_yaml_path.exists():
+        logger.warning(f"Found existing {flow_yaml_path.as_posix()}, will not respect it in runtime.")
+        with open(flow_yaml_path, "r", encoding=DEFAULT_ENCODING) as f:
+            existing_content = f.read()
+    with open(flow_yaml_path, "w", encoding=DEFAULT_ENCODING) as f:
+        dump_yaml({"entry": entry}, f)
```

Comment:
will we merge the exsiting flow yaml? or write a yaml with one line on entry?

Reply:
For flow test case, write a yaml with one line on entry